flash/core/classification.py

```diff
@@ -121,7 +120,8 @@ def __init__(self, multi_label: bool = False, threshold: float = 0.5):
 
     def serialize(self, sample: Any) -> Union[int, List[int]]:
         sample = sample[DefaultDataKeys.PREDS] if isinstance(sample, Dict) else sample
-        sample = torch.tensor(sample)
+        if not torch.is_tensor(sample):
```

Comment:
```suggestion
        if not isinstance(sample, torch.Tensor):
```

does the same thing under the hood. PyTorch people said that this is legacy from when tensors where not first class classes in python and was a bad design decision.

Reply:
Interesting, didn't know that, always thought `is_tensor` was best practice, changing now :smiley: